config_src/drivers/solo_driver/MOM_surface_forcing.F90

```diff
@@ -1542,6 +1550,94 @@ subroutine buoyancy_forcing_linear(sfc_state, fluxes, day, dt, G, US, CS)
   call callTree_leave("buoyancy_forcing_linear")
 end subroutine buoyancy_forcing_linear
 
+
+! Sets the necessary MARBL forcings via the data override facility.
+subroutine MARBL_forcing_from_data_override(fluxes, day, G, US, CS)
+  type(forcing),            intent(inout) :: fluxes !< A structure containing thermodynamic forcing fields
+  type(time_type),          intent(in)    :: day    !< The time of the fluxes
+  type(ocean_grid_type),    intent(inout) :: G      !< The ocean's grid structure
+  type(unit_scale_type),    intent(in)    :: US     !< A dimensional unit scaling type
+  type(surface_forcing_CS), pointer       :: CS     !< pointer to control structure returned by
+                                                    !! a previous surface_forcing_init call
+  ! Local variables
+  real, pointer, dimension(:,:) :: atm_co2_prog         =>NULL() !< Prognostic atmospheric CO2 concentration [ppm]
+  real, pointer, dimension(:,:) :: atm_co2_diag         =>NULL() !< Diagnostic atmospheric CO2 concentration [ppm]
+  real, pointer, dimension(:,:) :: atm_fine_dust_flux   =>NULL() !< Fine dust flux from atmosphere [kg/m^2/s ~> RZ/T]
```

Comment:
The description of the units for `atm_fine_dust_flux` and the next 7 variables uses syntax that is the reverse of that used elsewhere in the MOM6 code with the units as used by the model coming first, then their unscaled equivalent.  Moreover the double-divide is ambiguous here where the expression should be considered to be left-associative or right-associative, changing its meaning.

  Please change these 7 unit descriptions to `[R Z T-1 ~> kg m-2 s-1]`.

Reply:
Addressed in 5402ce0